src/Repository/index.js

```diff
@@ -1,16 +1,25 @@
-const { ResourceNotFoundError } = require('../utils/errors')
+const {
+    ResourceNotFoundError,
+    AllContributorBotError,
+} = require('../utils/errors')
 
 class Repository {
-    constructor({ repo, owner, github }) {
+    constructor({ repo, owner, github, defaultBranch }) {
         this.github = github
         this.repo = repo
         this.owner = owner
+        this.defaultBranch = defaultBranch
+        this.basedBranch
     }
 
     getFullname() {
         return `${this.owner}/${this.repo}`
     }
 
+    setBasedBranch(branchName) {
+        this.basedBranch = branchName
+    }
+
     async getFile(filePath) {
```

Comment:
I wonder how we can take the latest master file and overwrite the current branch :think: like a rebase

Reply:
We probably shouldn't address this here. This is probably a whole ticket in itself. #62.
I'm interested in hearing how you would solve this problem tho.